egs2/stop/asr1_multitask/conf/decode_asr_ar.yaml

```diff
@@ -0,0 +1,8 @@
+beam_size: 1
+ctc_weight: 0.0
+lm_weight: 0.
+maxlenratio: -20
+minlenratio: 0.
+penalty: 0.
+lang_prompt_token: "<|ar|> <|scr|> <|ar_scr|>"
+prompt_token_file: /projects/bbjs/arora1/new_download/espnet/egs2/stop/asr3_less_combined2_more_multilingual_data2/add_tokens-Copy1.txt
```

Comment:
Why this is an absolute path?
Many config files under this directory are an absolute path.